racetime/views/__init__.py

```diff
@@ -47,6 +47,8 @@
     Undisqualify,
     AddMonitor,
     RemoveMonitor,
+    RaceChatActions,
```

Comment:
As far as I can tell, since a URL is needed to be called by the form I need to set up a route in urls.py for both the view I created with the delete button for the chat that embeds next to the messages, and the actual delete message method. Without these lines I cannot even launch the server.

```
File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/vcunningham/.local/src/racetime-app/racetime/urls.py", line 57, in <module>
    path('chat_actions', views.RaceChatActions.as_view(), name='chat_actions'),
AttributeError: module 'racetime.views' has no attribute 'RaceChatActions'
```
[full error](https://paste.ee/p/ExlXC)

If there is a better way to do this that doesn't require including these classes here I'm all for it.

Reply:
Ah right sorry, I didn't see that it was being used in urls.py. Ignore me then, this is perfectly fine. 😁